### PR TITLE
Set-up goreleaser

### PR DESCRIPTION
**Pull request by isimluk**

### CODE REVIEW

File: .goreleaser.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+project_name: kubectl-falcon
+release:
+  github:
```
Comment (shawndwells) [inline .goreleaser.yml:3]: Update to CrowdStrike's repo

**Review by shawndwells — CHANGES_REQUESTED** [review]

File: .goreleaser.yml (hunk 1 above)
Comment (isimluk) [reply .goreleaser.yml:3]: oh, did I just squashed commits instead deleting them 🤦 

Good catch!

File: .goreleaser.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    owner: crowdstrike
+    name: kubectl-falcon
+builds:
+  - id: falcon
+    goos:
+    - linux
+    goarch:
+    - amd64
```
Comment (shawndwells) [inline .goreleaser.yml:11]: Does it make sense to also release ARM variant? Eg for the new macbooks and also for k8s on ARM? ACK'ing and merging this PR -- something to consider for future work.

**Review by shawndwells — APPROVED** [review]

File: .goreleaser.yml (hunk 2 above)
Comment (isimluk) [reply .goreleaser.yml:11]: Other variants will definitely come. However, it requires non-trivial effort to set-up CGO=1 release cross build in the CI. Future stuff.

**Pull request merged**